deploy/kubernetes/helm/che/values.yaml

```diff
@@ -89,6 +89,9 @@ global:
   chePluginRegistryUrlFormat: "plugin-registry-%s.%s"
 
 che:
+  properties:
```

Comment:
Why not ?
`  properties: {}`

Reply:
This fix was applied by https://github.com/eclipse/che/pull/15825/commits/da9c72db8bcb58a908b2dfe733d81680621e82fc and its comment is:

```
This is just a example for discussing configMapHelper. Do not apply this patch to the master branch.
```

That commit is just for explaining the usage. It must be reverted before merged. 
I agree `properties: {}` will be the best choice.

Reply:
@monaka 
Could you pls fix that?  I would like to merge the PR )

Reply:
@tolusha Thanks for heads up. I found a minor bug in this PR.
I'll fix it within a few days and will re-request review.

Reply:
@monaka 
Can we helm with anything?